src/sim/Types.hh

```diff
@@ -0,0 +1,23 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file Types.hh
+//---------------------------------------------------------------------------//
+#ifndef sim_Types_hh
+#define sim_Types_hh
```

Comment:
Oops, I missed that this was a holdover from me creating the file before we started using `#pragma once`.